src/v2/Apps/Consign/Components/SellWithArtsy.tsx

```diff
@@ -13,9 +13,12 @@ export const SellWithArtsy: React.FC = () => {
           </Text>
           <Text variant="text">Selling art differently</Text>
           <Spacer mt={6} />
-
-          {/* FIXME: todo link */}
-          <Button>Download the app</Button>
+          <Link
+            href="https://apps.apple.com/us/app/artsy-buy-sell-original-art/id703796080"
```

Comment:
This brings up a question in my mind about tracking app downloads - will bring up in Slack to loop in the rest of the team.